service/src/main/kotlin/fi/espoo/evaka/invoicing/data/VoucherValueDecisionQueries.kt

```diff
@@ -209,7 +209,7 @@ fun Database.Read.searchValueDecisions(
         "financeDecisionHandlerId" to financeDecisionHandlerId
     )
 
-    val (freeTextQuery, freeTextParams) = freeTextSearchQuery(listOf("head", "partner", "child"), searchTerms)
+    val (freeTextQuery, freeTextParams) = freeTextSearchQuery(listOf("head", "child"), searchTerms)
```

Comment:
There was no `partner` table in the SQL query anymore, that's why the search was broken.

Reply:
Why not include partner in the query instead of taking it off the free text search altogether?

Reply:
Not sure what's the correct thing to do 🤔 Because the search was broken, I assume that `partner` was there but has been removed since.

Reply:
I remember wondering why `partner` wasn't in that list and adding it at some point, but I guess I didn't realize that it wasn't in the actual query. I think `partner` should be there.

Reply:
Added partner and a test that checks that partner's details can be used for searching.